Test failing locally: test_parallel_calculation

## Expected behavior ##

No failed test

## Actual behavior ##

platform darwin -- Python 3.8.2, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: /Users/abhishandy/Code/mdanalysis/fork/testsuite, inifile: setup.cfg
plugins: forked-1.1.2, xdist-1.31.0, hypothesis-5.6.0

```python
___________________________________________________________________________________ TestEncore.test_parallel_calculation ___________________________________________________________________________________
[gw0] darwin -- Python 3.8.2 /Users/abhishandy/opt/miniconda3/envs/mda/bin/python

self = <MDAnalysisTests.analysis.test_encore.TestEncore object at 0x133984400>

    @pytest.mark.xfail(os.name == 'nt',
                       strict=True,
                       reason="Not yet supported on Windows.")
    def test_parallel_calculation(self):
    
        def function(x):
            return x**2
    
        arguments = [tuple([i]) for i in np.arange(0,100)]
    
        parallel_calculation = encore.utils.ParallelCalculation(function=function,
                                                                n_jobs=4,
                                                                args=arguments)
>       results = parallel_calculation.run()

analysis/test_encore.py:132: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../package/MDAnalysis/analysis/encore/utils.py:315: in run
    w.start()
../../../../../opt/miniconda3/envs/mda/lib/python3.8/multiprocessing/process.py:121: in start
    self._popen = self._Popen(self)
../../../../../opt/miniconda3/envs/mda/lib/python3.8/multiprocessing/context.py:224: in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
../../../../../opt/miniconda3/envs/mda/lib/python3.8/multiprocessing/context.py:283: in _Popen
    return Popen(process_obj)
../../../../../opt/miniconda3/envs/mda/lib/python3.8/multiprocessing/popen_spawn_posix.py:32: in __init__
    super().__init__(process_obj)
../../../../../opt/miniconda3/envs/mda/lib/python3.8/multiprocessing/popen_fork.py:19: in __init__
    self._launch(process_obj)
../../../../../opt/miniconda3/envs/mda/lib/python3.8/multiprocessing/popen_spawn_posix.py:47: in _launch
    reduction.dump(process_obj, fp)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

obj = <Process name='Process-2' parent=32939 initial>, file = <_io.BytesIO object at 0x139388180>, protocol = None

    def dump(obj, file, protocol=None):
        '''Replacement for pickle.dump() using ForkingPickler.'''
>       ForkingPickler(file, protocol).dump(obj)
E       AttributeError: Can't pickle local object 'TestEncore.test_parallel_calculation.<locals>.function'

../../../../../opt/miniconda3/envs/mda/lib/python3.8/multiprocessing/reduction.py:60: AttributeError
====================================================== 1 failed, 15980 passed, 121 skipped, 1 xfailed, 2 xpassed, 33071 warnings in 691.83s (0:11:31) ======================================================
```

## Code to reproduce the behavior ##

`pytest --disable-pytest-warnings --numprocesses 4`

## Current version of MDAnalysis ##

- Which version are you using? **0.20.1**
- Which version of Python? **3.8.2**
- Which operating system? **MacOS Catalina 10.15.3**

--- a/testsuite/MDAnalysisTests/analysis/test_encore.py
+++ b/testsuite/MDAnalysisTests/analysis/test_encore.py
@@ -47,7 +47,6 @@ def function(x):
     return x**2
 
 class TestEncore(object):
-
     @pytest.fixture(scope='class')
     def ens1_template(self):
         template = mda.Universe(PSF, DCD)

--- a/testsuite/MDAnalysisTests/analysis/test_encore.py
+++ b/testsuite/MDAnalysisTests/analysis/test_encore.py
@@ -43,7 +43,11 @@ import MDAnalysis.analysis.align as align
 import MDAnalysis.analysis.encore.confdistmatrix as confdistmatrix
 
 
+def function(x):
+    return x**2
+
 class TestEncore(object):
+
     @pytest.fixture(scope='class')
     def ens1_template(self):
         template = mda.Universe(PSF, DCD)
@@ -120,9 +124,6 @@ inconsistent results")
                        strict=True,
                        reason="Not yet supported on Windows.")
     def test_parallel_calculation(self):
-
-        def function(x):
-            return x**2
 
         arguments = [tuple([i]) for i in np.arange(0,100)]
 

--- a/testsuite/MDAnalysisTests/analysis/test_encore.py
+++ b/testsuite/MDAnalysisTests/analysis/test_encore.py
@@ -120,7 +120,6 @@ class TestEncore(object):
 inconsistent results")
 
     @pytest.mark.xfail(os.name == 'nt',
-                       strict=True,
                        reason="Not yet supported on Windows.")
     def test_parallel_calculation(self):
 

